server/controllers/index.js

```diff
@@ -22,9 +22,13 @@ router.post('/auth/login', (req, res) => {
         const user = {
             username: req.body.username
         };
-        currentUsers.push(user);
-        gameLogic.joinGame(user);
-        res.sendStatus(200);
+        if (gameLogic.canJoinGame()) {
+            currentUsers.push(user);
+            gameLogic.joinGame(user);
+            res.sendStatus(200);
+        } else {
+            res.sendStatus(403);
```

Comment:
I think 403 is more for 'you don't have the permission level for this request'?

Reply:
https://httpstatusdogs.com/403-forbidden

Reply:
I dunno, I've seen different definitions in different places, on the Mozilla website it says:
403 Forbidden
The client does not have access rights to the content, i.e. they are unauthorized, so server is rejecting to give proper response. Unlike 401, the client's identity is known to the server.

Reply:
For custom errors maybe we should just use a 400 bad request with a body containing the reason for the error? e.g. 400 "Error: the game has already started" @Ro4052 thoughts on status code use?